win32ss/user/winsrv/consrv/frontends/gui/conwnd.c

```diff
@@ -2164,6 +2164,20 @@ GuiConsoleHandleScrollbarMenu(VOID)
 }
 */
 
+HBITMAP
+CreateFrameBufferBitmap(HDC hDC, int width, int height)
+{
+    BITMAPINFO bmi;
+    ZeroMemory(&bmi, sizeof(BITMAPINFO));
+    bmi.bmiHeader.biSize = sizeof(BITMAPINFOHEADER);
+    bmi.bmiHeader.biWidth = width;
+    bmi.bmiHeader.biHeight = height;
+    bmi.bmiHeader.biPlanes = 1;
+    bmi.bmiHeader.biBitCount = GetDeviceCaps(hDC, BITSPIXEL);
+    bmi.bmiHeader.biCompression = BI_RGB;
+    return CreateDIBSection(NULL, &bmi, DIB_RGB_COLORS, NULL, NULL, 0);
```

Comment:
4th parameter may be non-NULL.

Reply:
Yes, but it's optional and we don't need it.